cypress/support/pages/course/pageCommonCourses.js

```diff
@@ -2,10 +2,18 @@
 
 class Courses_Common {
 
+  static #createCourse = '[data-testid="add-course-button"]'
+  static #createContent = '[data-testid="add-content-button"]'
   static #toolsTab = '[data-testid="tools"]'
   static #toolsList = '[data-testid="course_tool_list_add_tool"]'
   static #courseOverviewNavigationButton = '[data-testid="Course-Overview"]'
   static #addNewToolButton = '[data-testid="add_new_tool"]'
+  static #newTask = '[data-testid="fab_button_add_task"]'
+  static #elements = '[class="elements"]'
```

Comment:
also in this element we can put data-testid

Reply:
As discussed we don't need this element anymore.